### PR TITLE
Add CC-BY-SA copyright notice to first page

### PR DESCRIPTION
**Pull request by kfogel**

### CODE REVIEW

File: whitepaper/dsnp_whitepaper.tex [hunk 1]
```diff
@@ -38,6 +48,10 @@
 
 \maketitle
 
+\standalonefootnote{\copyright\, 2020 Liberty 3.0 LLC.\\
```
Comment (wilwade) [inline whitepaper/dsnp_whitepaper.tex:51]: @sbendar Should this be Unfinished LLC or perhaps something else?

Comment (kfogel) [reply whitepaper/dsnp_whitepaper.tex:51]: Ah, yeah, thanks @wilwade -- I was taking a guess, but obviously please adjust as needed.

Comment (kfogel) [reply whitepaper/dsnp_whitepaper.tex:51]: (Most of my time went to LaTeX research, as you might guess :-) .)

Comment (sbendar) [reply whitepaper/dsnp_whitepaper.tex:51]: Confirmed that we will go with UnfinishedLabs LLC until we transfer the rights to the DNSP foundation.

**Comment by wilwade** [comment]
I wasn't 100% sure I liked using the footnote style, so I tested another option with just putting it at the bottom and starting the paper on "page 2" (I think we would want to make the second page be page 1 instead and remove the page number from the title page).

Thoughts? @kfogel / @sbendar / @harry-evans 

PR version:
image (3)


Alt version:
image (2)

**Comment by kfogel** [comment]
FWIW, I like your way (that is, the second version) better, @wilwade.

Agreed about title page not having a page number and the first page of body text being page 1, too, unless there's some convention that says it should be the other way (I don't know of any such convention, but maybe others who are more experienced in academic publication would know).

**Comment by wilwade** [comment]
@kfogel 
1. Can you turn on permission to edit your branch? (How to: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)

2. Re: the only reason I can think of to have the "second page" still be page number 2 (after removing the page number from the title page) would be to have the pdf page numbering line up with the numbering in the paper. Thoughts?

**Comment by kfogel** [comment]
@wilwade

I tried following GitHub's instructions, but they must be out of date -- there seems to be no setting where I can control this, and in fact the PR is not even listed on the upstream repository side, it's only in the destination repository.  Go ahead and just make a new PR from my branch or do whatever you need to do.

I think it's best for PDF pages and page numbers to match if possible, so yeah, I'd vote for the numbering starting on the "second page" with "2".  But it's not a big deal either way -- your call!

**Comment by wilwade** [comment]
Replaced by https://github.com/LibertyDSNP/papers/pull/7

**Pull request closed without merging**